doc/users/whats_new.rst

```diff
@@ -24,6 +24,55 @@ revision, see the :ref:`github-stats`.
 New in Matplotlib 2.1
 =====================
 
+Documentation
++++++++++++++
+
+The examples have been migrated to use `sphinx gallery
+<https://sphinx-gallery.readthedocs.io/en/latest/>`__.  This allows
+better mixing of prose and code in the examples, provides links to
+download the examples as both a Python script and a Jupyter notebook,
+and improves the thumbnail galleries.  The examples have been
+re-organized into :ref:`tutorials` and a :ref:`gallery`.
+
+Many docstrings and examples have been clarified and improved.
+
+
+New features
+++++++++++++
+
+String categorical values
+-------------------------
+
+All plotting functions now support string categorical values as input.
+For example:
+
+.. plot::
+   :include-source:
+   :align: center
+
+   data = {'apples':10, 'oranges':15, 'lemons':5, 'limes':20}
```

Comment:
could be PEP8-'ed... (space after colon)

Reply:
Also (I guess it's kind of too late for this now...) I am *really* surprised that this does not keep key order (dict implicitly maintain key order on 3.6, but even passing an OrderedDict does not help)